lug/config.yaml

```diff
@@ -1,6 +1,6 @@
 interval: 10 # Interval between pollings
 loglevel: 4 # 1-5
-concurrent_limit: 5
+concurrent_limit: 1
```

Comment:
Limit 1 is okay for normal sync, you may change this back. But for bootstrapping, we're cloning in local area network, the speed is fast enough.